lib/passport-saml/saml.js

```diff
@@ -1019,7 +993,7 @@ SAML.prototype.validateSignatureForRedirect = function (urlString, signature, al
   function hasMatch (ourAlgo) {
     // The incoming algorithm is forwarded as a URL.
     // We trim everything before the last # get something we can compare to the Node.js list
-    var algFromURI = alg.toLowerCase().replace(/.*#(.*)$/,'$1');
+    const algFromURI = alg.toLowerCase().replace(/.*#(.*)$/,'$1');
```

Comment:
Not sure when const, and when var? Is var substitute for let?

Reply:
It seems that the upstream has changed their coding conventions and has been partially migrating to the es6 conventions such as const and arrow functions.  

I either have no idea why all the variables aren't `const`s or `let`s (for mutable references if there's any) in this commit for example, but I hope the style will be consistent in the future. 
https://github.com/vrk-kpa/suomifi-passport-saml/pull/15/commits/53e7363567123f9099ecc6163f148df3a79b1314